### DISCUSSION TITLE
Any link is provided between subject BMI and X-ray data?

### DISCUSSION THREAD

**Discussion by AmudhaTK** [MIT-LCP/mimic-code#1330] [MIMIC-IV-CXR] [answered]
In MIMIX-CXR dataset, information about radiology image  and radiological report was provided , but patients information is not available in that, is  there any way to find the corresponding clinical information of the patients?

**Comment by marymlucas** [comment, marked as answer]
Hi @AmudhaTK in the DICOM header from MIMIC-CXR the PatientID corresponds directly to the subject_id so I generally use this to find the patient and their clinical information for the stay (by matching StudyDate with the hospital dates in MIMIC-IV). Hope this helps.

PS: I found the tutorials by @alistairewj linked to [here ](https://mimic.mit.edu/docs/iv/tutorials/video/) really helpful in tying all the tables together

**Reply by AmudhaTK** [reply]
Thank you Marymlucas. i got it.
